vale/RTD/sso.yaml

```
@@ -0,0 +1,9 @@
+extends: substitution
+message: Use 'Single sign-on' instead of '%s'
```

Also note the message double substitution is important. Very silly this isn't just a default for the `message` attr.

```suggestion
message: Use '%s' instead of '%s'
```